artiq/firmware/satman/main.rs

```diff
@@ -447,6 +447,19 @@ const SI5324_SETTINGS: si5324::FrequencySettings
     crystal_ref: true
 };
 
+#[cfg(all(has_si5324, rtio_frequency = "100.0"))]
+const SI5324_SETTINGS: si5324::FrequencySettings
+    = si5324::FrequencySettings {
+    n1_hs  : 5,
+    nc1_ls : 10,
+    n2_hs  : 10,
+    n2_ls  : 250,
+    n31    : 50,
+    n32    : 50,
+    bwsel  : 4,
+    crystal_ref: true
```

Comment:
How did you determine these?

Reply:
I used DSPLL sim program from Silicon Laboratories (which miraculously worked under Wine, more or less) - put in the input clock freq (100MHz from siphaser), desired output, and it calculated these values for me. I verified that they're mapped by the driver to proper register values as well.

Reply:
OK great